src/vs/workbench/parts/explorers/media/treeExplorer.contribution.css

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@octref please check for typical CSS we use in trees. For example the items currently are not looking centered because they miss a `line-height: 22px` instruction. 

![image](https://cloud.githubusercontent.com/assets/900690/19959225/6484bf0a-a1a5-11e6-89c6-e991c6cf9c7f.png)